lms/views/lti_launches.py

```diff
@@ -153,6 +154,7 @@ def should_launch(request):
 
 @view_config(route_name='lti_launches', request_method='POST')
 @lti_launch()
+@create_h_user
```

Comment:
This decorator is placed _after_ `@lti_launch()`. `@lti_launch()` is the decorator that verifies the signature on the LTI request and aborts if it's invalid, so we won't attempt to create h users for unsigned LTI requests.

Reply:
Educational moment: Why is `@lti_launch()` invoked while `@create_h_user` is a reference?

Reply:
It's because [lti_launch()](https://github.com/hypothesis/lms/blob/6e39b7e1e15be5df4048dcdf8b70fc1eff071461/lms/util/lti_launch.py#L25-L65) is actually a function that returns a decorator. A function that returns a function that returns a function. A "decorator factory".

It has been implemented this way because the top-level `lti_launch()` function, the function that returns the decorator, actually takes a couple of arguments, so you can call it with different arguments (@lti_launch(...)`) and it'll return differently-configured versions of the decorator.

It never _is_ called with arguments though, always uses the default values...